e2e/scenarios/basic/res/hsl-conf/metadata.xml

```diff
@@ -0,0 +1,59 @@
+This XML file does not appear to have any style information associated with it. The document tree is shown below.
```

Comment:
```suggestion
```

This line can be removed. It is not the actual content of the XML, instead it is just a warning shown by the browser that has been copied and pasted by mistake.